.github/workflows/publish-docs.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           pip install --disable-pip-version-check -r docs/requirements_pip.txt
           pip --version
           pip install --disable-pip-version-check -r docs/requirements.txt
-          go install github.com/k0sproject/version/cmd/k0s_sort@v0.4.1
+          go install github.com/k0sproject/version/cmd/k0s_sort@v0.4.2
```

Comment:
maybe just use @latest

Reply:
I'd rather stay at fixed version numbers, so that builds won't break because of incompatible version changes.